CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Presto
+
+Presto is an Apache 2.0-licensed open source distributed SQL query engine for running interactive analytic queries against data sources of all sizes, ranging from gigabytes to petabytes.
+
+Presto was designed and written from the ground up for interactive analytics and approaches the speed of commercial data warehouses while scaling to the size of organizations like Facebook.
+
+The community owned and driven Presto project is supported by the [Presto Foundation](https://prestodb.io), an independent nonprofit organization with open and neutral governance, hosted under the [Linux Foundation](https://linuxfoundation.org). Learn more about the [Presto Foundation](https://prestodb.io) here.
+
+All Presto development occurs on [GitHub](https://github.com/prestodb), following open source best practices.
+
+## Technical governance
+
+Presto follows industry best practices, with the technical project and its governance distinctly separate from the Presto Foundation.  The [Technical Charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md) establishes the Presto Technical Steering Committee as the body which oversees all technical aspects of the open source project.  By contrast, the [Presto Foundation](https://prestodb.io) Governing Board is responsible for raising and distributing funds to support project activities, and setting policy.  While the two groups are separate, they work in close coordination to establish, guide, and support activities that lead to a long-term, sustainable future for Presto.
+
+### Project roles and responsibilities
```

Comment:
We have another roles & responsibilities doc on wiki [here](https://github.com/prestodb/presto/wiki/Presto-Open-Source-Community-Roles-&-Responsibilities). Can you please make sure they are aligned?

Reply:
Thank you, I hadn't seen this. Do you have a sense whether it makes sense to maintain this in a wiki, or in the CONTRIBUTING.md file?

Reply:
`CONTRIBUTING.md` also makes sense to me. How are other projects typically maintaining it?

Reply:
In an email thread, Nezih, Chris, and I discussed adding this to the agenda for the upcoming TSC meeting, but I'm documenting that discussion here for anybody else who may have the same question.

Generally speaking, when policy changes require review and approval (through consensus or voting), we recommend using .md files in the project repo.  The main advantage is that changes are made via pull request, so there is both visibility and an audit trail.  The wiki (I believe?) is open for any github user to edit, so there's a risk of non-project members making changes that could fundamentally alter a policy without sufficient review.

This will be a good thing to discuss in the meeting (https://github.com/prestodb/tsc/blob/master/meetings/2019-11-19.md)